requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.9
```

Comment:
Diffs in this file should be removed , there is no change in `requirements.in`. How come that this is updated?

Reply:
prometheus-flask-exporter was moved from requirements.in to requirements-web.in.  Regenerated.